example/transformer/ag_news/README.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Evaluate Hugging Face Transformer `textattack/albert-base-v2-ag-news` on AG News dataset
+---
+
+```bash
+    git clone https://github.com/star-whale/starwhale.git
```

Comment:
```
    git clone https://github.com/star-whale/starwhale.git
    cd starwhale/client
    python3 -m pip install -e .
```
-->
```
   pip install starwhale
```
use release starwhale pypi package as the user example ?

Reply:
> ```
>     git clone https://github.com/star-whale/starwhale.git
>     cd starwhale/client
>     python3 -m pip install -e .
> ```
> 
> -->
> 
> ```
>    pip install starwhale
> ```
> 
> use release starwhale pypi package as the user example ?

This readme directs users to build dataset from code and to build model from code. How could we guarantee the compatibility if user installl a release version of starwhale.

Reply:
dev version is not recommended for the users.